netlogo-core/src/main/fileformat/NLogoFormat.scala

```diff
@@ -79,6 +79,10 @@ trait AbstractNLogoFormat[A <: ModelFormat[Array[String], A]] extends ModelForma
         val sectionLines = sections.getOrElse(name, Array[String]())
         if (sectionLines.isEmpty)
           "\n"
+        else if (name == "org.nlogo.modelsection.version"){
+          if(!is3DFormat) sectionLines.map(_.replace("3D ", "")).mkString("\n", "\n", "\n")
```

Comment:
This line seems to only check if the format is not 3D and convert the version to 2D, but doesn't convert in the other way (2D -> 3D).